src/js/chem_h_subtheme.behaviors.js

```diff
@@ -29,6 +29,17 @@
       let queryParam = new URLSearchParams(window.location.search);
       const searchResultsForm = $('#block-chem-h-subtheme-search-form .su-site-search__input');
       searchResultsForm.val(queryParam.get('key'));
+
+      // Person node header profile button
+      $('.su-quote__bio', context).append($('.su-person-profile-link', context).detach());
+
+      // Square image for Person lists
```

Comment:
@jdwjdwjdw Can you remove this comment if you don't need it?

Reply:
Updated, thanks for noticing that!